.setup/bin/setup_sample_courses.py

```diff
@@ -980,6 +1019,47 @@ def create(self):
                                         src = os.path.join(gradeable.lichen_sample_path, gradeable.plagiarism_submissions.pop())
                                         dst = os.path.join(submission_path, str(version))
                                         create_gradeable_submission(src, dst)
+                                elif gradeable.annotated_pdf is True:
+                                    if version == versions_to_submit:
+                                        annotation_version_path = os.path.join(annotation_path, str(versions_to_submit))
+                                        if not os.path.exists(annotation_version_path):
+                                            os.makedirs(annotation_version_path)
+                                            os.system("chown -R submitty_php:{}_tas_www {}".format(self.code, annotation_version_path))
```

Comment:
`os.chown()` might be better than `os.system()` here.

Reply:
I was kinda confused about this part because `os.chown()` requires uid and gid, and in this case it seems a specified group is needed. I basically looked at other parts of this file when I wrote this line.

Reply:
Given that this line appears elsewhere in this script, and it's already a pretty hacky script which only ever gets run in the development environment, it probably isn't worth the effort to change I suppose.